### ISSUE TITLE
Great Administrator can place improvement Citadel which has no stats, preventing unit automation!

### ISSUE THREAD

**Issue by denemcam** [SpacedOutChicken/DeCiv-Redux#15]
Hello and thank you for your hard work!

The issue was on the latest patch, 3.18.9-p1. Player and AI-controlled units weren't going anywhere or exploring on their own.

**Comment by SpacedOutChicken** [comment]
Regarding the Great Administrator: that unit can still place a Settlement, so unless I'm mistaken, the AI can still make some use of them. Just not as much use as a human player can. (This is unfortunately common in DeCiv; the mod has all kinds of interesting featurs that the AI is too dumb to use.)

Regarding the units not moving: was it because there were enemy units nearby? I know that my own units, when set to explore, will freeze up when they run into barbarian units.

**Comment by denemcam** [comment]
Here's how it happened: I played marathon, everything was good until I realised that AI empires weren't expanding or even building tile improvements (my workers were manually controlled). They still were able to build and use military units, only things related to automation seems broke. Barbarians weren't affected. 
I believe that happened because of the Citadel improvement was built. 
Ah, yes, and the explore freeze on enemy met is little bit annoying too.
